presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/MetastoreUtil.java

```diff
@@ -165,6 +165,9 @@ private static Properties getHiveSchema(
             }
         }
 
+        schema.putIfAbsent("hive.serialization.extend.nesting.levels", "true");
+        schema.putIfAbsent("hive.serialization.extend.additional.nesting.levels", "true");
+
```

Comment:
This isn't the best place, but https://github.com/prestosql/presto/blob/5217f63283c7cacbb39107fe363cb3039d78be02/presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java#L410-L425 is not called for writing.
Also there is this: https://github.com/prestosql/presto/blob/89eab99bb83aef1f469f6de6e6958b8012d15a5b/presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java#L1402-L1403